public/help/help.md

```diff
@@ -30,3 +31,65 @@ The following procedure describes adding and updating settings of custom tracks.
+
+   A track JSON object might look like this:
+   ```
+   {
+      "trackFile": "exampleData/internal/snp1kg-BRCA1.vg.xg",
+      "trackType": "graph",
+      "trackColorSettings": {
+         "mainPalette": "greys",
+         "auxPalette": "ygreys",
+         "colorReadsByMappingQuality": false
+      }
+   }
+   ```
+
+   * The `trackFile` is the path (from the server working directory) or URL (any HTTP/HTTPS URL) to the track file 
+     Examples:
+      * `tracks[0][trackFile]=exampleData/internal/snp1kg-BRCA1.vg.xg`
+      * `tracks[0][trackFile]=https://public.gi.ucsc.edu/~anovak/graphs/trivial-brca1.vg`
+   * The `trackType` specifies the type of the track, which can be "graph", "haplotype", "read"
+      * Example: `tracks[0][trackType]=graph`
+   * The `trackColorSettings` provides color information and is an optional setting.
+      * The color palettes are `mainPalette` and `auxPalette`. 
+         * The colors for the palettes can be hex codes starting with a `#`, or premade palettes: `greys`, `ygreys`, `blues`, `reds`, `plainColors`, or `lightColors`. 
+         * The palettes are used differently for different track types
+            * For graphs, the `mainPalette` colors the primary reference path while `auxPalette` is used for the other paths. 
+            * For haplotypes, only the `mainPalette` is used. 
+            - For reads, the `mainPalette` colors the forward-strand reads and the `auxPalette` colors the backward-strand reads. 
+      * `colorReadsByMappingQuality` is a boolean value that determines if reads are colored based on their mapping quality or not. It is an optional field that has a default of `false`. 
+      * Examples
+         * `tracks[0][trackColorSettings][mainPalette]=greys`
+         * `tracks[0][trackColorSettings][auxPalette]=ygreys`
+         * `colorReadsByMappingQuality=true`
+2. `region`
+   This is a region input that is documented at step 3 of
+   [Displaying Visualizations](#displaying-visualizations). This region will be loaded in the tubemap visualization once the link is followed.
+   * Example: `region=17:3A1-100`
+3. `bedfile`
+   The `bedFile` is the path (from the server working directory) or URL (any HTTP/HTTPS URL) to the bed file. It is an optional field. More information on bedfile structure and creation is documented [here](https://github.com/vgteam/sequenceTubeMap?tab=readme-ov-file#preparing-subgraphs-in-advance). 
+   * Examples:
+      * `bedFile=exampleData/internal/snp1kg-BRCA1.bed`
+      * `bedfile=https://raw.githubusercontent.com/vgteam/sequenceTubeMap/ca4f2485231ee4182173bec19489ba940b27461a/exampleData/cactus.bed`
+      * `bedfile=none`
+4. `datatype`
+   Describes type of data as `built-in`, `mounted files`, or synthetic `examples`.
+      * `built-in`: If the `datatype` field is set to `built-in`, you should set the `name` field to the name of a preset defined in `DATA_SOURCES` in `config.json`.
```

Comment:
But what does this really mean, though? We should explain the value like we explain the fields, with a noun phrase before we explain additional semantics or give advice.

Also, maybe we should call them "data sources" and not "presets", since the config file does.

```suggestion
      * `built-in`: Predefined data sources from the server configuration file. If the `datatype` field is set to `built-in`, you should set the `name` field to the `name` of a data source defined in `DATA_SOURCES` in `config.json`.
```